assets/js/blocks/featured-category/edit.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+
+export const Edit = ( props: unknown ): JSX.Element => {
```

Comment:
I haven't worked with blocks much, so forgive me for asking: isn't there a way to know this type at all?

Reply:
> I haven't worked with blocks much, so forgive me for asking: isn't there a way to know this type at all?
Yes, we can try to type it, but without the right typeguards, there is the risk that the code will be less safe than now.